source/extensions/filters/network/zookeeper_proxy/filter.cc

```diff
@@ -348,6 +351,9 @@ void ZooKeeperFilter::onResponse(const OpCodes opcode, const int32_t xid, const
     break;
   }
 
+  config_->scope_.histogram(fmt::format("{}.{}_latency", config_->stat_prefix_, opname))
```

Comment:
If this lands after https://github.com/envoyproxy/envoy/pull/7810/files you might want to use the new way of referencing stat names